src/puptoo/process/profile.py

```diff
@@ -644,6 +647,22 @@ def system_profile(
     if gcp_network_interfaces:
         profile["public_ipv4_addresses"] = _remove_empty_string(gcp_network_interfaces.public_ips)
 
+    if bootc_status:
+        try:
+            profile["bootc_status"] = {}
+            # Another filed "cachedUpdate" could be added if required later
+            status = bootc_status.get('status', {})
+            for bootc_key in ["booted", "staged", "rollback"]:
```

Comment:
As per the schema under inventory-repo, we have not considered [cachedUpdate](https://github.com/RedHatInsights/inventory-schemas/blob/master/schemas/system_profile/v1.yaml#L678C11-L678C23). Is there any reason?

Reply:
Hi @r14chandra , yes, for the "cachedUpdate",  we are skipping it at this stage, referring to the `Description` part  of this [RHINENG-7612](https://issues.redhat.com/browse/RHINENG-7612) by our initiator's:

> There is a 'cachedUpdate' field that I am still gathering info on. It is OK to skip populating that for now and we can revisit it later.

Reply:
Thanks for confirming @JoySnow.